### PR TITLE
Update style and workshops

### PR DESCRIPTION
**Pull request by JulioPDX**
- Modified accept words
- Disabled Google exclamation check
- Removed extra plurals
- Removed double space
- Add alex to catch insensitive and inconsiderate writing

### CODE REVIEW

File: workshops/ansible.md [hunk 1]
```diff
@@ -504,7 +504,7 @@ To do this, we'll review a playbook together. Specifically, the **`~/project/lab
 
 ![Ansible Playbook Anatomy](assets/images/ansible_playbook_anatomy.png)
 
-At the very start of our playbook, we have the ==Play==. This is the very root of the playbook. it is where we define the Managed Nodes we'd like to target with this play, as well as the list of task(s) we'd like
+At the very start of our playbook, we have the ==Play==. This is the very root of the playbook. it is where we define the Managed Nodes we'd like to target with this play, as well as the list of task we'd like
```
Comment (mthiel117) [inline workshops/ansible.md]: I this should be `tasks` vs `task`

**Comment by mthiel117** [comment]
Since `repo` is going to be an allowed word.  Can you update this branch to undo the changes where it changed `repo` to `repository`?

After that it looks good to me to merge.

**Comment by JulioPDX** [comment]
> Since `repo` is going to be an allowed word. Can you update this branch to undo the changes where it changed `repo` to `repository`?
> 
> After that it looks good to me to merge.

yes, give me a moment

**Review by mthiel117 — APPROVED** [review]
LGTM....

**Pull request merged**